common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -43,6 +43,7 @@
 
     public enum Feature {
         ACCOUNT2(Type.EXPERIMENTAL),
+        DEVICE_ACTIVITY(Type.EXPERIMENTAL),
```

Comment:
It was my understanding that we were concerned with the device history part so we made it experimental.  Since we have put that feature off, do we still need Device Activity to be experimental?

Reply:
I don't have any objection to removing this feature and related checks.

Reply:
@pedroigor Can we go ahead and remove that before we merge this PR?  

If keycloak.profile.feature.device_activity=enabled is not used then Device Activity UI will crash.  Right now it assumes that you have at least one device and one session.  If you are using the UI then that will always be true.